Squared/rooms/rm_main/rm_main.yy

```diff
@@ -87,12 +87,12 @@
     },
     "roomSettings":     {
         "id": "4535478a-1cb4-45d7-9867-137443a79f1b",
-        "Height": 576,
+        "Height": 768,
```

Comment:
We were planning on having smaller rooms, that's why that one was 576 X 678. @afrey15 and I thought there was too much wasted space in these rooms. Can you change them back?